DanTup.BrowserSelector/DanTup.BrowserSelector.csproj

```diff
@@ -57,14 +60,17 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="BrowserSelector.ini">
+    <EmbeddedResource Include="BrowserSelector.ini">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
+    </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
     <None Include="Browser.ico" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>if exist "%25Bin%25\apps\$(TargetName)\$(TargetFileName)" copy /Y /V /B "$(TargetPath)" "%25Bin%25\apps\$(TargetName)\$(TargetFileName)"</PostBuildEvent>
```

Comment:
I don't really understand what this is for (or why the path is "%Bin%\apps\"; nothing else has an "apps" folder?)